src/Subscription.php

```diff
@@ -643,6 +643,46 @@ public function nextPayment()
         return new Payment($payment['amount'], $payment['currency'], $payment['date']);
     }
 
+    /**
+     * Begin creating a new modifier.
+     *
+     * @param  float  $amount
+     * @return \Laravel\Paddle\ModifierBuilder
+     */
+    public function newModifier($amount)
+    {
+        return new ModifierBuilder($this, $amount);
+    }
+
+    /**
+     * Get all of the modifiers for this subscription.
+     *
+     * @return \Illuminate\Support\Collection
+     */
+    public function modifiers()
+    {
+        $result = Cashier::post('/subscription/modifiers', array_merge([
```

Comment:
Does Paddle paginate that endpoint?

Reply:
Nope: https://developer.paddle.com/api-reference/subscription-api/modifiers/listmodifiers